### PR TITLE
make downstream task

### PR DESCRIPTION
**Pull request by miseulkim**

### CODE REVIEW

File: modules/model.py [hunk 1]
```diff
@@ -14,6 +14,15 @@ def __init__(self, hp, device):
         self.nclass = self.hp['model']['n_symbols']
         self.fc     = nn.Linear(self.enc_hidden_dim, self.nclass)
         
-    def forward(self, wav_padded, wav_lengths, text_padded, text_lengths, criterion):
-        return self.model(wav_padded, mask = True)
-        
+    def forward(self, wav_padded, wav_lengths, text_padded, text_lengths, criterion, mask=True):
```
Comment (Yeona-Hong) [inline modules/model.py:17]: mask False일 때 logits에다가 projection layer 추가한 것 확인하였습니다

File: utils/data_utils.py [hunk 2]
```diff
@@ -39,8 +39,8 @@ def get_data_pair(self, path_data):
         path_wav = os.path.join(self.data_path, path_wav)
 
         wav, _ = sf.read(path_wav)
-        wav = wav[:((len(wav)//320) -1)* 320 + 80]
-        #print(wav.shape[0] / 320, wav.shape[0])
+        #wav = wav[:((len(wav)//320) -1)* 320 + 80]
+        wav = wav[:((len(wav)//200) -1) * 200]
```
Comment (Yeona-Hong) [inline utils/data_utils.py:43]: wav length를 바꾼 이유가 있나요?

File: modules/selectra.py [hunk 3]
```diff
@@ -115,9 +116,9 @@ def forward(self, x, padding_mask = None, mask = False):
         x_disc = self.discriminator(x_disc)
         x_disc = self.disc_proj(x_disc)
 
-        #disc_loss = F.cross_entropy(x_disc.transpose(2,1), mask_indices.long())
+        disc_loss = F.cross_entropy(x_disc.transpose(2,1), mask_indices.long())
 
-        return mlm_loss, mlm_loss#disc_loss
+        return mlm_loss, disc_loss
```
Comment (Yeona-Hong) [inline modules/selectra.py:121]: return 할 때 mlm_loss는 살려놔야하지 않아여?

File: utils/data_utils.py (hunk 2 above)
Comment (miseulkim) [reply utils/data_utils.py:43]: 사용하고 있는 pre-trained soundstream의 encoder의 resampling rate가 200hz이기 때문입니다!

File: modules/model.py (hunk 1 above)
Comment (miseulkim) [reply modules/model.py:17]: 확인 감사드립니다! ><

File: utils/data_utils.py (hunk 2 above)
Comment (Yeona-Hong) [reply utils/data_utils.py:43]: 넵 감사합니다!!
